.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "linux"]
 	path = linux
-	url = https://github.com/Ultimaker/linux
+	url = https://github.com/Ultimaker/linux.git
```

Comment:
the addition of .git seems odd to me....

Reply:
That;s the url, submodule usually just appends the .git underwater, but you can also provide it.